Ginger/Ginger/Drivers/DriversWindows/MobileDriverWindow.xaml.cs

```diff
@@ -1387,8 +1392,8 @@ private System.Windows.Point GetPointOnMobile(System.Windows.Point pointOnImage)
 
                 if (mDriver.GetDevicePlatformType() == eDevicePlatformType.iOS) // && mDriver.GetAppType() == eAppType.NativeHybride)
                 {
-                    ratio_X = (xDeviceScreenshotImage.Source.Width / 2) / xDeviceScreenshotImage.ActualWidth;
-                    ratio_Y = (xDeviceScreenshotImage.Source.Height / 2) / xDeviceScreenshotImage.ActualHeight;
+                    ratio_X = (xDeviceScreenshotImage.Source.Width) / xDeviceScreenshotImage.ActualWidth;
```

Comment:
Need to see if we need to add also flag for this for doing the changes only for needed devices after testing with iPad and others